src/main/java/org/gridsuite/network/map/dto/utils/ElementUtils.java

```diff
@@ -39,6 +41,39 @@ public static void setIfNotNan(@NonNull final DoubleConsumer setter, final doubl
         }
     }
 
+    private static ConnectablePosition.Feeder getFeederInfos(Identifiable<?> identifiable, int index) {
```

Comment:
This code should use ThreeSides
```suggestion
    private static ConnectablePosition.Feeder getFeederInfos(Identifiable<?> identifiable, ThreeSides side) {
```

Reply:
With the index we can handle injections and branches, but the ThreeSides type cannot manage that